packages/containers/containers.3.3/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "simon.cruanes.2007@m4x.org"
+synopsis: "A modular, clean and powerful extension of the OCaml standard library"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "build" "@doc" "-p" name ] {with-doc}
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test & ocaml:version < "4.11"}
```

Comment:
why is this restrained to such old versions?

Reply:
In the past we re-enabled them: https://github.com/ocaml/opam-repository/pull/16944/commits/a900dcf34b0e8c6757aa61d2d14341aaea3af934

Reply:
But did not do it for a number of intermediate releases I think

Reply:
ah good point. Thanks! I've done the same for this release and the previous ones

Reply:
hmm I didn't add this constraint, that I'm aware of? :thinking: